ee/server/service/mdm.go

```diff
@@ -1074,3 +1074,26 @@ func (svc *Service) mdmWindowsDisableOSUpdates(ctx context.Context, teamID *uint
 	err := svc.ds.DeleteMDMWindowsConfigProfileByTeamAndName(ctx, teamID, mdm.FleetWindowsOSUpdatesProfileName)
 	return ctxerr.Wrap(ctx, err, "delete Windows OS updates profile")
 }
+
+func (svc *Service) GetMDMManualEnrollmentProfile(ctx context.Context) ([]byte, error) {
+	if err := svc.authz.Authorize(ctx, &fleet.MDMAppleEnrollmentProfile{}, fleet.ActionRead); err != nil {
```

Comment:
I think this is super confusing (sorry about that) but this `MDMAppleEnrollmentProfile` is a different thing, and it's only available to global admins:

https://github.com/fleetdm/fleet/blob/e423bdc4b70bdcd6206e1bac65dbd0c0bcf693eb/server/authz/policy.rego#L655-L660

I think the only requirement for this is for the user to have been authenticated, we could either check that and authorize, or add a new autzh type in `policy.rego`

Reply:
Aha, I shoulda asked to confirm that that was the right object. I think we can just check if the user is authenticated.